### ISSUE TITLE
ERROR: "OverflowError" when trying to save model with to_disk

### ISSUE THREAD

**Issue by rogeriobromfman** [explosion/spacy-transformers#183]
I've fine-tuned BERT with my own corpus using huggingface/transformers. I now have a directory (`./bert`) with the following structure:

```
bert_config.json
optimizer.pt
pytorch_model.bin
scheduler.pt
training_args.bin
vocab.txt
```

I am now trying to convert this model to use with SpaCy, using the following code:

```
from spacy_transformers import TransformersLanguage, TransformersWordPiecer
from spacy_transformers import TransformersTok2Vec

base_dir = '/home/rogerio/bert'
nlp = TransformersLanguage(trf_name=base_dir, meta={"lang": "pt"})
nlp.add_pipe(nlp.create_pipe("sentencizer"))
nlp.add_pipe(TransformersWordPiecer.from_pretrained(nlp.vocab, base_dir))
nlp.add_pipe(TransformersTok2Vec.from_pretrained(nlp.vocab, base_dir))
nlp.to_disk('/home/rogerio/spacy')
```

I get an error on the last line saying:

```
Traceback (most recent call last):
  File "_packer.pyx", line 181, in srsly.msgpack._packer.Packer._pack
OverflowError: Python int too large to convert to C unsigned long

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/rogerio/env/lib/python3.6/site-packages/spacy/language.py", line 911, in to_disk
    util.to_disk(path, serializers, exclude)
  File "/home/rogerio/env/lib/python3.6/site-packages/spacy/util.py", line 645, in to_disk
    writer(path / key)
  File "/home/rogerio/env/lib/python3.6/site-packages/spacy/language.py", line 909, in <lambda>
    serializers[name] = lambda p, proc=proc: proc.to_disk(p, exclude=["vocab"])
  File "pipes.pyx", line 207, in spacy.pipeline.pipes.Pipe.to_disk
  File "/home/rogerio/env/lib/python3.6/site-packages/spacy/util.py", line 645, in to_disk
    writer(path / key)
  File "pipes.pyx", line 205, in spacy.pipeline.pipes.Pipe.to_disk.lambda6
  File "/home/rogerio/env/lib/python3.6/site-packages/spacy_transformers/_tokenizers.py", line 64, in to_bytes
    return srsly.msgpack_dumps(msg)
  File "/home/rogerio/env/lib/python3.6/site-packages/srsly/_msgpack_api.py", line 16, in msgpack_dumps
    return msgpack.dumps(data, use_bin_type=True)
  File "/home/rogerio/env/lib/python3.6/site-packages/srsly/msgpack/__init__.py", line 40, in packb
    return Packer(**kwargs).pack(o)
  File "_packer.pyx", line 285, in srsly.msgpack._packer.Packer.pack
  File "_packer.pyx", line 291, in srsly.msgpack._packer.Packer.pack
  File "_packer.pyx", line 288, in srsly.msgpack._packer.Packer.pack
  File "_packer.pyx", line 246, in srsly.msgpack._packer.Packer._pack
  File "_packer.pyx", line 192, in srsly.msgpack._packer.Packer._pack
OverflowError: Integer value out of range
```

**Comment by rogeriobromfman** [comment]
{UPDATE] Interestingly, when I ran the same code in Google Colab instead of my EC2 machine, it ran with no problems.

**Comment by dcshapiro** [comment]
same issue

**Comment by svlandeg** [comment]
This should be fixed in the recent publication of v1.0 so I think this can be closed.

New bugs can be opened on the [spaCy GitHub tracker](https://github.com/explosion/spaCy/issues) and feature requests or ask for coding help can be posted on the [spaCy discussions forum](https://github.com/explosion/spaCy/discussions).

Thanks for testing the early prereleases of this package! All the feedback has been really helpful :-)
